### PR TITLE
feat(plasma-tokens): Update `asdk` theme with xml

### PR DESCRIPTION
**Pull request by neretin-trike**
В существующую схему `asdk` добавлены значения градиентов для android в формате XML.

:baby_chick: Download canary assets:

[color_asdk_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_ios-swift--canary.567.5398343479.swift)  
[color_asdk_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_kotlin--canary.567.5398343479.kt)  
[color_asdk_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_react-native--canary.567.5398343479.ts)  
[color_asdk_xml--canary.567.5398343479.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_xml--canary.567.5398343479.xml)  
[color_sberHealth_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.567.5398343479.swift)  
[color_sberHealth_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.567.5398343479.kt)  
[color_sberHealth_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.567.5398343479.ts)  
[color_sberHealth_xml--canary.567.5398343479.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.567.5398343479.xml)  
[color_sbermarket_business_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.567.5398343479.swift)  
[color_sbermarket_business_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.567.5398343479.kt)  
[color_sbermarket_business_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.567.5398343479.ts)  
[color_sbermarket_business_xml--canary.567.5398343479.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.567.5398343479.xml)  
[color_sbermarket_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.567.5398343479.swift)  
[color_sbermarket_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.567.5398343479.kt)  
[color_sbermarket_metro_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.567.5398343479.swift)  
[color_sbermarket_metro_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.567.5398343479.kt)  
[color_sbermarket_metro_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.567.5398343479.ts)  
[color_sbermarket_metro_xml--canary.567.5398343479.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.567.5398343479.xml)  
[color_sbermarket_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.567.5398343479.ts)  
[color_sbermarket_selgros_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.567.5398343479.swift)  
[color_sbermarket_selgros_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.567.5398343479.kt)  
[color_sbermarket_selgros_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.567.5398343479.ts)  
[color_sbermarket_selgros_xml--canary.567.5398343479.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.567.5398343479.xml)  
[color_sbermarket_wlbusiness_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.567.5398343479.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.567.5398343479.kt)  
[color_sbermarket_wlbusiness_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.567.5398343479.ts)  
[color_sbermarket_wlbusiness_xml--canary.567.5398343479.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.567.5398343479.xml)  
[color_sbermarket_xml--canary.567.5398343479.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.567.5398343479.xml)  
[color_sberonline_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.567.5398343479.swift)  
[color_sberonline_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.567.5398343479.kt)  
[color_sberonline_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.567.5398343479.ts)  
[color_sberonline_xml--canary.567.5398343479.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.567.5398343479.xml)  
[color_sberprime_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.567.5398343479.swift)  
[color_sberprime_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.567.5398343479.kt)  
[color_sberprime_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.567.5398343479.ts)  
[color_sberprime_xml--canary.567.5398343479.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.567.5398343479.xml)  
[shadow_sbermarket_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.567.5398343479.ts)  
[typo_mage_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.567.5398343479.swift)  
[typo_mage_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.567.5398343479.kt)  
[typo_mage_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.567.5398343479.ts)  
[typo_plasma_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.567.5398343479.swift)  
[typo_plasma_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.567.5398343479.kt)  
[typo_plasma_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.567.5398343479.ts)  
[typo_ruler_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.567.5398343479.swift)  
[typo_ruler_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.567.5398343479.kt)  
[typo_ruler_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.567.5398343479.ts)  
[typo_sage_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.567.5398343479.swift)  
[typo_sage_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.567.5398343479.kt)  
[typo_sage_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.567.5398343479.ts)  
[typo_sbermarket_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.567.5398343479.swift)  
[typo_sbermarket_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.567.5398343479.kt)  
[typo_sbermarket_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.567.5398343479.ts)  
[typo_soulmate_ios-swift--canary.567.5398343479.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.567.5398343479.swift)  
[typo_soulmate_kotlin--canary.567.5398343479.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.567.5398343479.kt)  
[typo_soulmate_react-native--canary.567.5398343479.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.567.5398343479.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.218.0-canary.567.5398343479.0
  npm install @salutejs/cra-template-plasma-basic-template@2.4.0-canary.567.5398343479.0
  npm install @salutejs/plasma-colors@0.8.0-canary.567.5398343479.0
  npm install @salutejs/plasma-core@1.123.0-canary.567.5398343479.0
  npm install @salutejs/plasma-hope@1.218.0-canary.567.5398343479.0
  npm install @salutejs/plasma-icons@1.152.0-canary.567.5398343479.0
  npm install @salutejs/cra-template-plasma-shop-template@2.3.0-canary.567.5398343479.0
  npm install @salutejs/plasma-temple@1.167.0-canary.567.5398343479.0
  npm install @salutejs/plasma-tokens-b2b@1.23.0-canary.567.5398343479.0
  npm install @salutejs/plasma-tokens-b2c@0.32.0-canary.567.5398343479.0
  npm install @salutejs/plasma-tokens-core@0.3.0-canary.567.5398343479.0
  npm install @salutejs/plasma-tokens-web@1.38.0-canary.567.5398343479.0
  npm install @salutejs/plasma-tokens@1.57.0-canary.567.5398343479.0
  npm install @salutejs/plasma-typo@0.35.0-canary.567.5398343479.0
  npm install @salutejs/plasma-ui@1.199.0-canary.567.5398343479.0
  npm install @salutejs/plasma-web@1.218.0-canary.567.5398343479.0
  npm install @salutejs/plasma-cy-utils@0.64.0-canary.567.5398343479.0
  npm install @salutejs/plasma-sb-utils@0.121.0-canary.567.5398343479.0
  npm install @salutejs/plasma-tokens-utils@0.28.0-canary.567.5398343479.0
  # or 
  yarn add @salutejs/plasma-b2c@1.218.0-canary.567.5398343479.0
  yarn add @salutejs/cra-template-plasma-basic-template@2.4.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-colors@0.8.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-core@1.123.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-hope@1.218.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-icons@1.152.0-canary.567.5398343479.0
  yarn add @salutejs/cra-template-plasma-shop-template@2.3.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-temple@1.167.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-tokens-b2b@1.23.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-tokens-b2c@0.32.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-tokens-core@0.3.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-tokens-web@1.38.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-tokens@1.57.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-typo@0.35.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-ui@1.199.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-web@1.218.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-cy-utils@0.64.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-sb-utils@0.121.0-canary.567.5398343479.0
  yarn add @salutejs/plasma-tokens-utils@0.28.0-canary.567.5398343479.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-567/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-567/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-567/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-567/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-567/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-567/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-567/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-567/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-567/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-567/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-567/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-567/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

Top 5 significant render changes:
packages/plasma-ui/src/components/Pickers/Pickers.perftest.tsx#Default: -12.49%

File: packages/plasma-tokens/data/themes/asdk.json [hunk 1]
```diff
@@ -685,6 +813,14 @@
                                 "locations": [0, 0.65],
                                 "startPoint": { "x": 0.7, "y": 0.96 },
                                 "endPoint": { "x": 0.3, "y": 0.04 }
+                            },
+                            "xml": {
+                                "kind": "gradient",
+                                "type": ".linear",
```
Comment (Yakutoc) [inline packages/plasma-tokens/data/themes/asdk.json:819]: @neretin-trike 

Из любопытства, а почему именно такая запись `.linear` с точкой?

**Review by Yakutoc — APPROVED** [review]

**Review by Yeti-or — APPROVED** [review]

File: utils/plasma-tokens-native/config.ts [hunk 2]
```diff
@@ -7,27 +7,38 @@ import {
...
-const tokensTypoPath = 'properties/typo/';
-const tokensShadowPath = 'properties/shadow/';
+const styles = {
+    color: {
+        path: 'properties/color/',
+        platforms: ['ios-swift', 'react-native', 'kotlin', 'xml'],
+    },
+    typo: {
+        path: 'properties/typo/',
+        platforms: ['ios-swift', 'react-native', 'kotlin'],
```
[the remark this answers is not in the document]
Comment (Yeti-or) [reply utils/plasma-tokens-native/config.ts:28]: это на прекрасное будущее

File: utils/plasma-tokens-native/config.ts [hunk 3]
```diff
@@ -71,11 +82,22 @@ const getStyleDictionaryColorConfig = (brand: string): Config => ({
                 },
             ],
         },
+        xml: {
+            transforms: ['attribute/cti', 'name/cti/snake', 'color/hex8android', 'color/gradientXML'],
```
[the remark this answers is not in the document]
Comment (Yeti-or) [reply utils/plasma-tokens-native/config.ts:86]: https://amzn.github.io/style-dictionary/#/transforms

**Review by Yeti-or — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-567/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-567/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-567/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-567/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-567/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-567/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-567/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-567/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-567/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-567/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-567/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-567/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-567/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-567/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-567/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-567/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-567/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-567/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-567/temple-storybook/

**Pull request merged**